index.html

```diff
@@ -112,6 +112,59 @@ <h2>A project for first-time contributions</h2>
           </div>
         </div>
         <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card END ________  -->
+<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
Doh! 😅 Looks like you accidentally pasted in an extra copy of the template!

No worries, this happens to the best of us. Can you please remove it so that other future contributors (like yourself!) aren't confused?